src/config/test_config.py

```diff
@@ -1,6 +1,7 @@
 # Disable the test codebuild jobs to be run
-DISABLE_SANITY_TESTS = False
+DISABLE_SANITY_TESTS = True
```

Comment:
Do revert these change before merging in

Reply:
Yes, these are meant to be reverted.

Reply:
fixed